examples/go-client/api-admin.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   # allowed for the users of this role. By default nothing is allowed.
   allow:
     rules:
+      - resources: ['cert_authority']
```

Comment:
Can you expand who would need this resource and is there ever a chance of not needing all the crud?